docs/notebooks/pulse_efficient_transpilation_qsim.ipynb

```diff
@@ -0,0 +1,765 @@
qrI3AgAAQJlZFwMAAAAsu+aiAwCg8y67/Qe57dFLcsYRN2XFXqsUHQ4AAABAVdkbAQAAoMysiwEAAAA6xjfSAdSpv786KaOuPS5rrrpBvnHu7kmS1m49cs6I+wqODAAAAKDr2RsBAACgzKyLAQAAADrOQTqAOrX6Kmvnpu+3FR0GAAAAQCHsjQAAAFBm1sUAAAAAHdfU1tZmRwVoGHNnJbedXXQUdNTuI5KW1uW/jt9/farU7z9J2tqSebMrc61qae6eNDVV7nptbcmsuZW7XjW0tlTuM5ADlF09jgEqOw/UYw5U+v2XuQ7WgrLnYNGsiepTJddEcqA+WReXe12sFyheI/UC9fj501g5mNRfHjbi519PdTBRC2tBI40Da6L6ZF1MJXOgaOpA8eqtH9ILFa+RxkA9fv40Vg4m9ZmHjfY7KFK91eFELa4FxiBQBN9IBwA0jKamxvlHls5qakp6lLjDkwOUnTFA2XOg7HWwFpQ9BwGKZh7WDxRNDhbL508tkIfFUgeLZwwAlJs6UDz9ULGMgWL5/KkF8rDc1OHiGYMAy6a56AAAAAAAAAAAAAAAAAAAoCs5SAcAAAAAAAAAAAAAAABAQ3OQDgAAAAAAAAAAAAAAAICG5iAdAAAAAAAAAAAAAAAAAA3NQToAAAAAAAAAAAAAAAAAGpqDdAAAAAAAAAAAAAAAAAA0NAfpAAAAAAAAAAAAAAAAAGhoDtIBAAAAAAAAAAAAAAAA0NAcpAMAAAAAAAAAAAAAAACgoTlIBwAAAAAAAAAAAAAAAEBDc5AOAAAAAAAAAAAAAAAAgIbmIB0AAAAAAAAAAAAAAAAADc1BOgAAAAAAAAAAAAAAAAAamoN0AAAAAAAAAAAAAAAAADQ0B+kAAAAAAAAAAAAAAAAAaGgO0gEAAAAAAAAAAAAAAADQ0BykAwAAAAAAAAAAAAAAAKChOUgHAAAAAAAAAAAAAAAAQEPrVnQAAACV0taWzJtddBQd09w9aWqq3PXa2pJZcyt3vWpobancZyAHKLt6HANUdh6oxxyo9Psvcx2sBWXPQYCimYfrrx/QCxSvkXqBevz8aawcTOovDxvx86+nOpiohbWg0cYBQJHUgeLVWz+kFypeI42Bevz8aawcTOozDxvtd1CkeqvDiVpcC4xBoAgO0gEADWPe7OS2s4uOomN2H5G0tFbuerPmJieOrtz1quGMQ5IeFepK5QBlV49jgMrOA/WYA5V8/2Wvg7Wg7DkIUDTzcP31A3qB4jVSL1CPnz+NlYNJ/eVho33+9VYHE7WwFjTaOAAokjpQvHrrh/RCxWukMVCPnz+NlYNJfeZho/0OilRvdThRi2uBMQgUobnoAAAAAAAAAAAAAAAAAACgKzlIBwAAAAAAAAAAAAAAAEBDc5AOAAAAAAAAAAAAAAAAgIbmIB0AAAAAAAAAAAAAAAAADc1BOgAAAAAAAAAAAAAAAAAamoN0AAAAAAAAAAAAAAAAADS0bkUHAABQtMeeHZNvnLt7u9t6tq6QtVcfmGFbHZ4Dd/pKWlq0TY1MDgBlZx6kaHIQoFjmYYomB6HczAGUnTEAgFpA2RkDUG7mACiecQiUjRkNAOBfdt/ik9l2k33Tlra88s9puemhi3LuNcfm+b89la9/7BdFh0cVyAGg7MyDFE0OAhTLPEzR5CCUmzmAsjMGAFALKDtjAMrNHADFMw6BsnCQDgDgXzbuv1WGDT1swc/77/ilfH7kJvnj/eflcx/6TlZZcfUCo6Ma5ABQduZBiiYHAYplHqZochDKzRxA2RkDAKgFlJ0xAOVmDoDiGYdAWThIBwCwGL1aV8gm622fOx+/LFNeerZ0C8EfH9ZUsWt99ddtFbtWNZU9BwDKPg+qhcUrew4CFK3s87BeoHhlz0EoO3NAsdTB4hkDAKgFxdIPFc8YgHIzB6AWF884BBqVg3QAAEsw9aVnkyR9eq9acCTV9Y/pz2eXQ3+Qrfb5etGhFK6sOQDwtrLOg2ph7ShrDgLUirLOw3qB2lHWHATmMwcUQx2sHcYAAGpBMfRDtcMYgHIzB5SXWlw7jEOgETlIBwDwLzNnv5nX3pietra2vPLPabnm3nMzfvIj2WSdbbP26gOLDq+q/vrINdlgy/2LDqPq5ABQdubBd5S1FhZNDgIUyzz8Dr1AMeQglJs5oHaog8UwBgBQC2qHfqgYxgCUmzmAd1OLi2EcAmVRyoN006dPz8iRI3PFFVdk0qRJWX311TN8+PCcfvrpGTFiRH75y1/mnHPOyTHHHFN0qABAFV1046m56MZT292286Dh+cpBPy0oouK8Om1chuz55aLDqDo5AJSdefAdZa2FRZODAMUyD79DL1AMOQjlZg6oHepgMYwBANSC2qEfKoYxAOVmDuDd1OJiGIdAWZTuIN2jjz6affbZJ9OmTcsKK6yQzTbbLFOmTMnZZ5+dZ599Ni+//HKSZIsttig2UKBLzJs3L1fe9eNc96dRmfbKxKyywurZdcjH85m9/zu9WlcoOjygYB/e7sjsOvjgzJk3O3+d+kRGjzkj01+blNbuPRc85okJd+bk8/dZ6Llz5s7KvHlzc8PIudUMuUvMmvl6WnuuVHQYhZADNLpLbv1uxk1+OOMmPZRpL/81733Pevn1yROLDosaYh6cr8y1sGhyELqevRGWxDw8n16gOHKw61kXUsvMAbVBHSyOMQBdz5qYWqcW1Ab9UHGMga5nX4BaZg7gbWpxcYxDoCxKdZBu+vTp2X///TNt2rQcd9xxOfXUU7PSSvML7ciRI3PiiSemW7duaWpqyuDBgwuOFugKP7/m67nqrrOz06CD8rHdjsvzLz6Vq+46O89OfiRnHHlzmpubiw4RKFD/vhtnq4HDkiTbbrJPBq2/c77+s53z48uPzimH/TZJ8v4Ndsk133m93fOmvzYlXz5763xkx8b4Ntvnn7gp675/r6LDKIQcoNH98o8nZ6Xeq2bj/lvljRmvFh0ONcg8OF+Za2HR5CB0PXsj
```

Comment:
any ideas why the variance here is so large?

Reply:
In estimatorV1, the standard deviation should be $\sqrt{variance}/\sqrt{shots}$ - the variance was large since it was not decided by $\sqrt{shots}$; fixed in the new version.